runtime/lua/vim/lsp/util.lua

```diff
@@ -75,7 +75,7 @@ function M.set_lines(lines, A, B, new_lines)
   if #prefix > 0 then
     lines[i_0] = prefix..lines[i_0]
   end
-  return lines
+  return lines, i_0, (i_0 + #new_lines - 1), prefix, suffix
```

Comment:
Do we actually use all of these returns?

Can you add documentation to this function to define what it returns as well.

Reply:
Currently, no using all of the return values.
But the values are `start` or `end` specific so I put the all values of pairs.

Reply:
I will add the documentation.

Reply:
Sorry, I should have mentioned the location to look at the style. Can you document it in the style of:

`:help dev-lua-doc` ?

I know not all of the functions are documented that way, but trying to work on moving toward that :)